www/docs/mp/index.php

```diff
@@ -383,10 +383,10 @@
 // is the party we're comparing this MP to different from the party they're currently in?
 $data['party_switcher'] = (slugify($data['current_party_comparison']) != slugify($data["comparison_party"]));
 
-// Do any necessary extra work based on the page type, and send for rendering.
+// Update the social image URL generation logic
 switch ($pagetype) {
-
     case 'votes':
+        $data['og_image'] = \MySociety\TheyWorkForYou\Url::generateSocialImageUrl($member_name, 'Voting Summaries', $data['current_assembly']);
```

Comment:
instead of repeating this we could set an `$og_image_subtitle` variable here and then after the switch have a single place that the `$data['og_image']` is set. Would also need to remove the special case setting this for single policy pages but as that link no longer works that seems fine.

Reply:
Not sure - the rendering happens inside the switch so not sure we can tidy it that way.

Reply:
oh, doh! so it does. ignore this.